src/ui.form-date-picker-range/configs/default.config.js

```diff
@@ -34,8 +34,8 @@ export default /*tw*/ {
   },
   periodsRow: "mb-1 flex min-w-64 gap-1",
   periodButton: `
-    flex h-[3.125rem] w-full cursor-pointer flex-col items-center justify-center rounded-lg bg-gray-100
-    px-1.5 py-2.5 text-center text-xs font-medium text-brand-900 hover:bg-brand-200
+    flex h-[3.125rem] w-full cursor-pointer flex-col items-center justify-center bg-gray-100
```

Comment:
cursor-pointer and other styles is needed for the button? Better o use button props.